app/assets/javascripts/mixins/mixin-responsive.js

```diff
@@ -0,0 +1,53 @@
+export default {
+    data () {
+      return {
+        windowWidth: 0,
+        currentBreakpoint: '',
+        breakpoints: {
+          small: 767, // MUST MATCH VARIABLES IN assets/stylesheets/_settings
+          medium: 1024,
+          large: 1200
```

Comment:
These will need to match the variables in css

Reply:
I actually ended up not needing this mixin at all so I've now removed it